lib/logstash/inputs/rabbitmq/march_hare.rb

```diff
@@ -100,6 +102,9 @@ def setup
 
       # exchange binding is optional for the input
       if @exchange
+        @ch.exchange(@exchange,
```

Comment:
good catch..looking at [bunny's implementation](https://github.com/logstash-plugins/logstash-input-rabbitmq/blob/master/lib/logstash/inputs/rabbitmq/bunny.rb#L96-L98), there's no exchange declaration.
Why is it needed here, @ekimekim ?